WinImageBuilder.psm1

```diff
@@ -1129,54 +985,48 @@ function New-WindowsOnlineImage {
         if ($cpuCount -eq 0) {
             $cpuCount = 1
         }
-        if ($CpuCores -gt $cpuCount) {
+        if ($CONFIG["cpu_count"] -gt $cpuCount) {
             Write-Warning "CpuCores larger then available (logical) CPU cores.
                 Setting CpuCores to $coreCount"
-            $CpuCores = $coreCount
+            $CONFIG["cpu_count"] = $coreCount

```

Comment:
We should try not to overwrite configuration values. throw an error or use an extra variable